hare/eligibility/oracle_test.go

```diff
@@ -202,15 +202,15 @@ func TestCalcEligibility_EligibleFromHareActiveSet(t *testing.T) {
 		"384460966938c87644987fe00c0f9d4f9a5e2dcd4bdc08392ed94203895ba325036725a22346e35aa707993babef716aa1b6b3dfc653a44cb23ac8f743cbbc3d": 1,
 		"15c5f565a75888970059b070bfaed1998a9d423ddac9f6af83da51db02149044ea6aeb86294341c7a950ac5de2855bbebc11cc28b02c08bc903e4cf41439717d": 1,
 	}
-	for hex, exp := range sigs {
-		sig := util.Hex2Bytes(hex)
+	for s, exp := range sigs {
+		sig, _ := hex.DecodeString(s)
```

Comment:
require no error?

Reply:
It wasn't handled before, since what's decoded here are constants anyway and will never error, but I added the `require.NoError` 🙂